README.md

```diff
@@ -4,6 +4,34 @@ A list of climate friendly extensions that can help you reduce the carbon footpr
 ## CO2 emissions trackers and green website raters
 
 ### Globemallow
+#### Description
+
+Use Globemallow to quickly view a sustainability grade and sustainability metrics for a website that you visit.
+Globemallow was created to help designers and developers build websites using more sustainable development practices.
+
+**The Sustainability Grade**
```

Comment:
After I merged it I noticed tiny little things that escaped my notice when just reading the raw markdown!

One of its gotchas I always fall for is forgetting that apart from headings, you need to leave one blank line for a normal linebreak. Here you did 1 linebreak but forgot the second, so there was no actual break. Need to add one more linebreak. 

Also I'd suggest using italics (single asterisks or underscores (`_`)?instead of double asterisks) so it doesn't look like the H4 headings.